### PR TITLE
Fix tests includes and remove VLD checks

### PR DESCRIPTION
**Pull request by dcristoloveanu**
Fix tests includes and remove VLD checks

### CODE REVIEW

File: tests/singlylinkedlist_ut/main.c [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#include <stddef.h>
```
Comment (anporumb) [inline tests/singlylinkedlist_ut/main.c]: missing copyright

**Comment by anporumb** [comment]
> // Copyright (c) Microsoft. All rights reserved.

not released under MIT?

---
Refers to: tests/srw_lock_ut/main.c:1 in 690865d. [](commit_id = 690865dff17faa75c6b94b7b986ee760b9acda49, deletion_comment = False)

File: tests/singlylinkedlist_ut/main.c (hunk 1 above)
Comment (dcristoloveanu) [reply tests/singlylinkedlist_ut/main.c]: Fixed.

---
In reply to: [291777946](https://github.com/Azure/azure-c-shared-utility/pull/331#discussion_r291777946) [](ancestors = 291777946)

**Comment by dcristoloveanu** [comment]
> // Copyright (c) Microsoft. All rights reserved.

Fixed.

---
In reply to: [500065737](https://github.com/Azure/azure-c-shared-utility/pull/331#issuecomment-500065737) [](ancestors = 500065737)

---
Refers to: tests/srw_lock_ut/main.c:1 in 690865d. [](commit_id = 690865dff17faa75c6b94b7b986ee760b9acda49, deletion_comment = False)

**Pull request merged**